lib/modules/manager/gleam/index.ts

```diff
@@ -0,0 +1,17 @@
+import { HexDatasource } from '../../datasource/hex';
+import * as hexVersioning from '../../versioning/hex';
+
+export const displayName = 'gleam';
+export const url = 'https://gleam.run/';
+
+export { extractPackageFile } from './extract';
+export { updateArtifacts } from './artifacts';
+export { getRangeStrategy } from './range';
+
+export const defaultConfig = {
+  fileMatch: ['^gleam.toml$'],
```

Comment:
Are you sure it can never be in a subdirectory?

Reply:
it looks like this was discussed previously in [#26767](https://github.com/renovatebot/renovate/pull/26767#discussion_r1460622829). while gleam.toml can only ever be in the project's root dir, the existing regex did not evaluate gleam monorepo's or nested projects. i don't see a reason why renovate should limit that here, so this has been fixed in [636b682]. (https://github.com/renovatebot/renovate/pull/30345/commits/636b682583bf098770f33b393f08a650baf393e5)